<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitGlWidgetWithHeader</class>
 <widget class="QWidget" name="OrbitGlWidgetWithHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="OrbitTreeView" name="treeView">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>552</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="OrbitGLWidget" name="openGLWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>treeView</zorder>
  <zorder>scrollArea</zorder>
  <zorder>openGLWidget</zorder>
  <zorder>openGLWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrbitGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>orbitglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitTreeView</class>
   <extends>QTreeView</extends>
   <header>orbittreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
